cluster/saltbase/README.md

```diff
@@ -0,0 +1,12 @@
+# SaltStack configuration
+
+This is the root of the SaltStack configuration for Kubernetes. A high
+level overview for the Kubernetes SaltStack configuration can be found [in the docs tree.](../../docs/salt.md)
+
+This SaltStack configuration currently applies to default
+configurations for Debian-on-GCE, Fedora-on-Vagrant, Ubuntu-on-AWS and
+Ubuntu-on-Azure. (That doesn't mean it can't be made to apply to an
+arbitrary configuration, but those are only in-tree supported OS/IaaS
+combinations supported today.) As you peruse the configuration, these
```

Comment:
"As you peruse" comes as a surprise&mdash;what am I perusing? I think a note in the first paragraph that you can dive into sub-directories (especially `salt/`) for lots more information would help people know where to look for further documentation. (I worry that a project newbie wouldn't know specifically to look in `cluster/saltbase/salt/` to find the detailed docs you just wrote!)